IndianTea.prefab

```diff
@@ -0,0 +1,30 @@
+{

```

Comment:
This file looks to be a duplicate of the recipe prefab. Please delete it :-)

You need to have two different prefabs to make this work:

1. `assets/prefabs/food/cooked/IndianTea.prefab` - the file you have included as `ItemIndianTea.prefab`. The definition should be purely for what Indian Tea is. See for instance [LemonTea.prefab](https://github.com/Terasology/Cooking/blob/master/assets/prefabs/food/cooked/LemonTea.prefab)
1. `assets/prefabs/recipe/cook/RecipeIndianTea.prefab` - the actual recipe, prefixed as `Recipe` to set it apart from the other `IndianTea.prefab`. While the prefab name doesn't technically matter this is a good way to organize them (two prefabs with the same name wouldn't work even if they do different things)